packages/api/src/telemetry/index.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  BasicTracerProvider,
+  SimpleSpanProcessor,
+  ConsoleSpanExporter,
+} from '@opentelemetry/sdk-trace-base'
+import { Resource } from '@opentelemetry/resources'
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-grpc'
+import {
+  LoggerProvider,
+  SimpleLogRecordProcessor,
+  ConsoleLogRecordExporter,
+} from '@opentelemetry/sdk-logs'
+import { OTLPLogsExporter } from '@opentelemetry/exporter-logs-otlp-grpc'
+
+import type { Options } from '../'
+import { getFaststoreTelemetryPlugin } from './useFaststoreTelemetry'
+import packageJson from '../../package.json'
+
+export type GetTelemetryOptions = {
+  mode?: 'verbose' | 'dev'
+  experimentalSendLogs?: boolean
+}
+
+const FASTSTORE_API_VERSION = packageJson.version
+
+// TODO: These urls are hardcoded for now, but they should be configurable via ENV variables
+// They are only acessible from within the VTEX network, so they are not a security risk
+const TRACE_COLLECTOR_URL = 'opentelemetry-collector.vtex.systems'
+const TRACE_COLLECTOR_URL_DEV = 'opentelemetry-collector-beta.vtex.systems'
+
+const LOG_COLLECTOR_URL = 'opentelemetry-collector.vtex.systems'
+
+export function getTelemetry(
+  APIOptions: Options,
+  telemetryOptions?: GetTelemetryOptions
+) {
+  const honeycombCollectorOptions = {
+    url:
+      telemetryOptions?.mode === 'dev'
+        ? TRACE_COLLECTOR_URL_DEV
+        : TRACE_COLLECTOR_URL,
+  }
+
+  const openSearchCollectorOptions = {
+    url: LOG_COLLECTOR_URL,
+  }
+
+  // Create a new tracer provider
+  const tracerProvider = new BasicTracerProvider({
+    resource: new Resource({
+      'service.name': 'faststore-api',
+      'service.version': FASTSTORE_API_VERSION,
+      'service.name_and_version': `faststore-api@${FASTSTORE_API_VERSION}`,
+      platform: APIOptions.platform,
+      [`${APIOptions.platform}.account`]: APIOptions.account,
+      [`${APIOptions.platform}.environment`]: APIOptions.environment,
+      // TODO: include the following properties in the logs
```

Comment:
why not include now?

Reply:
Because ideally they need to be included within the plugin (`useFaststoreTelemetry`), and I really don't want to do that now 😅 It involves passing new parameters and I want to take the time to organize that nicely. Also, logs are not working as of now, so it would make no difference

Reply:
so create a new issue for it, pls?

Reply:
I'll do it in Jira!